main/hiredis/APKBUILD

```diff
@@ -34,6 +35,7 @@ package() {
 	cd "$_builddir"
 
 	make PREFIX="$pkgdir"/usr install
+	sed -i -e 's:^libdir.*$:libdir=${exec_prefix}/lib:' -e 's:^includedir.*:includedir=${prefix}/include/hiredis:' $pkgdir/usr/lib/pkgconfig/hiredis.pc
```

Comment:
we normally prefer patches instead of sed, so we can detect if upsteram has fixed things.

There is also no explanation in commit message or in comment that explains why this is needed. Why can we not just leave it as it is?